README.md

```diff
@@ -52,6 +52,12 @@ DataValues Time has been written by the Wikidata team, as [Wikimedia Germany]
 
 ## Release notes
 
+### 0.8.3 (2016-03-16)
+* `YearMonthDayTimeParser` did not accept options. Now it forwards to IsoTimestampParser and uses its calendar model and precision detection.
```

Comment:
Added optional `ParserOptions` parameter to the `YearMonthDayTimeParser` constructor.